release/Magarena/scripts/Supernatural_Stamina.groovy

```diff
@@ -9,14 +29,14 @@ def Return = MagicAbility.getAbility("When SN dies, return it to the battlefield
                 POS_TARGET_CREATURE,
                 this,
                 "Until end of turn, target creature\$ gets +2/+0 " +
-                "and gains \"When this creature dies, return it to the battlefield tapped under its owner's control.\""
```

Comment:
`this creature` is the correct phrasing as it is in the oracle text. Please restore the word `this`.